README.md

```diff
@@ -1,2 +1,27 @@
 # joining-18f-app
 The 18F application form
+
+## Installing
+
+After you've cloned this repo and made a Virtualenv with Python 3.x, run:
+    
+    pip install -r requirements.txt
```

Comment:
Hmm. I think this is indeed very system-specific (see https://github.com/Homebrew/homebrew/issues/25752). I think folks who need to run pip3 will probably pick up on that-- I'd rather leave as-is.